src/main/java/com/angelbroking/smartapi/SmartConnect.java

```diff
@@ -1,727 +1,781 @@
 package com.angelbroking.smartapi;
 
-import java.io.IOException;
-import java.net.Proxy;
-import java.util.List;
-
+import com.angelbroking.smartapi.http.SessionExpiryHook;
+import com.angelbroking.smartapi.http.SmartAPIRequestHandler;
+import com.angelbroking.smartapi.http.exceptions.CustomException;
+import com.angelbroking.smartapi.http.exceptions.SmartAPIException;
+import com.angelbroking.smartapi.models.*;
+import com.angelbroking.smartapi.utils.Constants;
+import com.angelbroking.smartapi.utils.ResponseParser;
+import com.angelbroking.smartapi.utils.Validators;
+import lombok.Data;
 import org.apache.commons.codec.digest.DigestUtils;
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import com.angelbroking.smartapi.http.SessionExpiryHook;
-import com.angelbroking.smartapi.http.SmartAPIRequestHandler;
-import com.angelbroking.smartapi.http.exceptions.SmartAPIException;
-import com.angelbroking.smartapi.models.Gtt;
-import com.angelbroking.smartapi.models.GttParams;
-import com.angelbroking.smartapi.models.Order;
-import com.angelbroking.smartapi.models.OrderParams;
-import com.angelbroking.smartapi.models.TokenSet;
-import com.angelbroking.smartapi.models.User;
+import java.io.IOException;
+import java.net.Proxy;
+import java.util.List;
+import java.util.Optional;
 
+@Data

```

Comment:
Why do we need this? Ideally the request body for smartConnect and the methods should be segregated into 2 files.

Reply:
have segregated SmartConnect into 2 files SmartConnect and SmartConnectParams 
POJO methods have been moved to SmartConnectParams